src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs

```diff
@@ -2419,11 +2449,11 @@ private bool CutOrDeleteSelection(bool isCut)
                     // Even though `OverlapsWithEditableBuffer` and `CutOrDelete` is sufficient to 
                     // delete editable selection,  we still need to handle box selection 
                     // differently to move caret to appropiate location after deletion.
-                    bool isEditable = selection.Mode == TextSelectionMode.Stream 
-                                                            ? OverlapsWithEditableBuffer(selection.SelectedSpans) 
+                    bool isEditable = selection.Mode == TextSelectionMode.Stream
+                                                            ? OverlapsWithEditableBuffer(selection.SelectedSpans)
                                                             : ReduceBoxSelectionToEditableBox();
                     if (isEditable)
-                    {                               
+                    {
```

Comment:
So many formatting changes. Maybe consider breaking that into a separate commit for blame purposes?